src/game/scripts/vscripts/abilities/brawler_detection.lua

```diff
@@ -0,0 +1,16 @@
+function brawler_detection( keys )
+	local caster = keys.caster
+	if caster:PassivesDisabled() then return end
```

Comment:
@OtsoTurpeinen how is this supposed remove the stats? if you end the script before the buff is removed then they will keep the stats.